packages/landing/static/index.html

```diff
@@ -6,14 +6,12 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
 
-    <meta property="og:title" content="LINZ Basemaps"/>
-    <meta property="og:image" content="/basemaps-card.jpeg"/>
-    <meta property="og:description" content="NZ’s authoritative and open digital basemap service for LINZ and the public." /> 
+    <meta property="og:site_name" content="LINZ Basemaps" />
```

Comment:
Is this additional tag required?

Reply:
Im not sure its "required" but it was a common tag I found when looking for opengraph tags